chromium_src/components/external_intents/android/intercept_navigation_delegate_impl.cc

```diff
@@ -6,7 +6,7 @@
 #include <string>
 
 #include "brave/common/pref_names.h"
-#include "chrome/android/chrome_jni_headers/InterceptNavigationDelegateImpl_jni.h"
+#include "components/external_intents/android/jni_headers/InterceptNavigationDelegateImpl_jni.h"
```

Comment:
seems like there should be a dep change to go along with this?

Reply:
Addressed in https://github.com/brave/brave-core/commit/efe8a130ba7dc8967c3364ff8ee11d9c227898cf